src/app/components/ContentBanner/ImageItem.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ImageItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleOnLoad = this.handleOnLoad.bind(this);
+    this.handleOnError = this.handleOnError.bind(this);
+
+    this.state = {
+      loaded: false,
+    };
+  }
+
+  componentWillUpdate() {
+    if (!this.state.loaded) {
+      this.handleOnLoad();
+    }
+  }
+
+  /**
+   * @desc Calculates whether a large or small image should be rendered.
+   * @param {object} obj - An image object with uri values.
```

Comment:
Not sure if you care but missing some @param docs here.

Reply:
Updated. Thanks.